driver-reactive-streams/src/test/functional/com/mongodb/reactivestreams/client/unified/ChangeStreamsTest.java

```diff
@@ -19,18 +19,59 @@
 import com.mongodb.lang.Nullable;
 import org.bson.BsonArray;
 import org.bson.BsonDocument;
+import org.junit.After;
 import org.junit.runners.Parameterized;
 
 import java.io.IOException;
 import java.net.URISyntaxException;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.List;
+
+import static com.mongodb.reactivestreams.client.syncadapter.SyncMongoClient.disableSleep;
+import static com.mongodb.reactivestreams.client.syncadapter.SyncMongoClient.enableSleepAfterCursorOpen;
+import static org.junit.Assume.assumeFalse;
 
 public final class ChangeStreamsTest extends UnifiedReactiveStreamsTest {
+
+    @SuppressWarnings("ArraysAsListWithZeroOrOneArgument")
+    private static final List<String> ERROR_REQUIRED_FROM_CHANGE_STREAM_INITIALIZATION_TESTS =
+            Arrays.asList(
+                    "Test with document comment - pre 4.4"
+            );
+
+    private static final List<String> EVENT_SENSITIVE_TESTS =
+            Arrays.asList(
+                    "Test that comment is set on getMore",
+                    "Test that comment is not set on getMore - pre 4.4"
+            );
+
+    private static final List<String> CURSOR_OPEN_TIMING_SENSITIVE_TESTS =
+            Arrays.asList(
+                    "Test with document comment",
+                    "Test with document comment - pre 4.4",
+                    "Test with string comment",
+                    "Test that comment is set on getMore"
```

Comment:
Does this mean that tests mentioned here and in `ERROR_REQUIRED_FROM_CHANGE_STREAM_INITIALIZATION_TESTS`, `EVENT_SENSITIVE_TESTS` are incorrect and we ignore them until they are fixed?

Reply:
I don't consider the tests in this list to be incorrect.  `enableSleepAfterCursorOpen` is a workaround that we already apply to other reactive unified test runners, and it's just necessary due to the way that reactive streams work.  There may be other enhancements we could make to the unified test format definition that would make this unnecessary, but for now this is what we have.  

The tests in the other two lists are not incorrect per se, but we don't have a workaround right now that would let them run, so we just skip them for now.  There may be enhancements that we could request in the unified test format, or perhaps other workarounds that we could add into the reactive test runner.  I separate them into two lists as a way to (sort of) document the two different problems that we're encountering.

I will open JAVA ticket(s) to track the possibility of un-skipping both classes of tests.

Reply:
> I don't consider the tests in this list to be incorrect. enableSleepAfterCursorOpen is a workaround

Right, sorry, I meant to mention only skipped tests, but then accidentally mixed everything together.

Thank you for explaining.